janus_core/calculations/phonons.py

```diff
@@ -278,11 +284,27 @@ def __init__(
         self.temp_max = temp_max
         self.temp_step = temp_step
         self.force_consts_to_hdf5 = force_consts_to_hdf5
+        self.hdf5 = hdf5
         self.plot_to_file = plot_to_file
         self.write_results = write_results
         self.write_full = write_full
         self.enable_progress_bar = enable_progress_bar
 
+        # Handle deprecation
+        if force_consts_to_hdf5:
```

Comment:
```suggestion
        if force_consts_to_hdf5 is not None:
```
Otherwise it won't catch setting it to `False`.